docs/src/pages/docs/functions/logic-functions.md

```diff
@@ -308,6 +438,39 @@ function. Either the original or modified value will be returned depending on
 the result of the predicate. Check out [`unless`](#unless) for a negated version
 of this function.
 
+```javascript
+import when from 'crocks/logic/when'
+
+import constant from 'crocks/combinator/constant'
+
+// prod :: Number -> Number -> Number
+const prod = a => b => b * a
+
+when(constant(true), prod(2), 21)
+//=> 42
+
+when(constant(false), prod(2), 21)
+//=> 21
+
+// gt :: Number -> Number -> Boolean
+const gt = a => b => b > a
+
+// dec :: Number -> Number
+const dec = a => a - 1
+
+// dec :: Number -> Number
+const safeDec = when(gt(0), dec)
```

Comment:
```suggestion
const decPositiveNumber = when(gt(0), dec)
```

`safe` in docs usually connotes a Maybe, whereas in our case we can be explicit about the math we're doing.